### ISSUE TITLE
NVDA doesn't announce page footer & page header in LibreOffice Writer

### ISSUE THREAD

**Issue by peterpacult** [nvaccess/nvda#11762]
### Steps to reproduce:
1. Open Writer
2. To add a header to a page, choose Insert - Header, and then select the page style for the current page from the submenu
3. To add a footer to a page, choose Insert - Footer, and then select the page style for the current page from the submenu
4. Try to read the footer/header with the screenreader NVDA

### Actual behavior:
Footer/header is not announced by NVDA

### Expected behavior:
NVDA indicates header/footer

### System configuration
NVDA installed/portable/running from source:
installed

NVDA version:
2020.3

Windows version:
Edition Windows 10 Home
Version 1909
Operating system build 18363.1016

Name and version of other software in use when reproducing the issue:
LibreOffice
Version: 	7.0.2.2 (x64)
Build ID: 	8349ace3c3162073abd90d81fd06dcfb6b36b994
CPU threads: 	4; OS: Windows 10.0 Build 18363; UI render: Skia/Vulkan; VCL: win
Locale: 		de-DE (de_DE); UI: de-DE
Calc: 		threaded

Other information about your system:
/

**Comment by peterpacult** [comment]
This bug was also reported to the Document Foundation:
https://bugs.documentfoundation.org/show_bug.cgi?id=137498

**Comment by Adriani90** [comment]
This is working now in LO 25.2 with NVDA 2024.4 Rc.
If you are still having this issue, please comment and we can reopen.
